src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorCompactChat.kt

```diff
@@ -0,0 +1,86 @@
+package at.hannibal2.skyhanni.features.garden.visitor
+
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.RegexUtils.groupOrNull
+import at.hannibal2.skyhanni.utils.RegexUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.StringUtils.removeResets
+import kotlin.time.Duration.Companion.milliseconds
+
+object GardenVisitorCompactChat {
```

Comment:
Please use auto reformatting on that file (in IntelliJ: Ctrl + Alt + L)

Reply:
Fixed in [f679a8f](https://github.com/DavidArthurCole/SkyHanni/commit/f679a8fe566fa3fd8e2d0ea0e27c16af8655becb)!